scripts/ci/90_docker-express.sh

```diff
@@ -1,10 +1,13 @@
 #!/bin/bash
 set -euxo pipefail
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 
 # shellcheck disable=SC1091
 source "$DIR"/_common.sh
 
 # Building Docker images for ExpressJS-based apps
+if command -v podman &>/dev/null; then
```

Comment:
What if you have both `docker` and `podman`?
```suggestion
if { ! command -v docker &>/dev/null } && { command -v podman &>/dev/null }; then
```

Reply:
The `_docker.sh` script should maybe be modified to allow either. And Podman should implement the missing options (or at least not fail)